### DISCUSSION TITLE
Peer to peer

### DISCUSSION THREAD

**Discussion by smort82** [eduardogsilva/wireguard_webadmin#42] [General]
Is it possible to have two webadmin instances (at different sites) and connect them on the same wireguard network by both being each others peer?

**Comment by eduardogsilva** [comment]
Wireguard_webadmin is not designed to work as a vpn client, but if you need to do this, I would suggest the following:

on wireguard_webadmin A:
Configure a peer for B. You could also configure a different instance of wireguard server just for this connection.

on wireguard_webadmin B:
Manually install wireguard on your linux and configure as a client for A.

This would be enough for connecting both servers. Maybe if you need something more specific, you would need to adapt this scenario.

Cheers!
